api/geth_backend.go

```diff
@@ -363,6 +363,8 @@ func (b *GethStatusBackend) OverwriteNodeConfigValues(conf *params.NodeConfig, n
 		return nil, err
 	}
 
+	conf.Networks = n.Networks
```

Comment:
Following [description](https://pkg.go.dev/github.com/imdario/mergo@v0.3.12#Merge) for `Merge` function, Networks slice is not updated accordingly, so it remains the same as it was in `conf`, that's why I did that manually here.

Reply:
FYI, I am waiting on this PR: https://github.com/status-im/status-go/pull/2664
That should make that line redundant right?

Reply:
From the description it says `WithOverride will make merge override non-empty dst attributes with non-empty src attributes values.` since `conf.Networks` is not an empty array, I think it won't do the overwrite.